modules/gitops-configuring-argo-cd-oidc.adoc

```diff
@@ -0,0 +1,80 @@
+// Module is included in the following assemblies:
+//
+// * 
+
+[id="configuring-argo-cd-oidc_{context}"]
+= Configuring Argo CD OIDC
+
+To configure Argo CD OpenID Connect (OIDC), you must generate your client secret, encode it, and add it to your custom resource.
+
+.Prerequisites
+
+* You have obtained your client secret. 
+
+.Procedure
+
+. Store the client secret you generated.
+
+.. Encode the client secret in base64:
++
+[source,terminal]
+----
+$ echo -n '83083958-8ec6-47b0-a411-a8c55381fbd2' | base64
+----
+
+.. Edit the secret and add the base64 value to an `oidc.keycloak.clientSecret` key:
++
+[source,terminal]
+---- 
+$ oc edit secret openshift-gitops-secret -n <namespace>
+----
++
+.Example YAML of the secret
+[source,yaml]
+----
+yaml apiVersion: v1 
+kind: Secret 
+metadata: name: argocd-secret 
+data:
+  oidc.keycloak.clientSecret: ODMwODM5NTgtOGVjNi00N2IwLWE0MTEtYThjNTUzODFmYmQy …
+----
+
+. Edit the `argocd` custom resource and add the OIDC configuration to enable the Keycloak authentication:
++
+[source,terminal]
+----
+$ oc edit argocd -n argocd
```

Comment:
Can we avoid providing the namespace as argocd here ?
can this be changed to 
`oc edit argocd -n <>`

Reply:
Fixed.